VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/participant-waiting-room.component.ts

```diff
@@ -239,7 +228,9 @@ export class ParticipantWaitingRoomComponent extends WaitingRoomBaseDirective im
     }
 
     get canStartJoinConsultation() {
-        return !this.isWitness && !this.isObserver && !this.isInterpreter && !this.isInterpretee;
+        return (
+            !this.isWitness && !this.isObserver && (!this.participant.linked_participants || !this.participant.linked_participants.length)
```

Comment:
in the base class, there is `isOrHasWitnessLink()` which you could re-use here

Reply:
I agree with you - makes sense to use it. Look though:
![image](https://user-images.githubusercontent.com/1761344/111899268-5d601b00-8a23-11eb-87ad-acaaad577601.png)
I'm gonna remove that func & just use the one in the base everywhere

Reply:
sorry which func do you plan to remove? the `isWitness` or `isWitnessOrHasWitnessLink`

Reply:
isWitness